engine/server/web_server/web_controller.hpp

```diff
@@ -334,38 +334,38 @@ class WebController : public oatpp::web::server::api::ApiController {
     return createDtoResponse(Status::CODE_200, status_dto);
   }
 
-  // TODO: implement with corresponding function later.
-  ADD_CORS(DeleteRecordsByID)
+  ADD_CORS(DeleteRecordsByPK)
 
-  ENDPOINT("POST", "/api/{db_name}/data/delete", DeleteRecordsByID,
+  ENDPOINT("POST", "/api/{db_name}/data/delete", DeleteRecordsByPK,
            PATH(String, db_name, "db_name"),
-           BODY_DTO(Object<DeleteRecordsReqDto>, body)) {
+           BODY_STRING(String, body)) {
     auto dto = StatusDto::createShared();
-
-    if (!body->table) {
+    vectordb::Json requestBody;
+    requestBody.LoadFromString(body);
```

Comment:
add payload validation check.
```suggestion
    auto valid = requestBody.LoadFromString(body);
    if (!valid) {
      status_dto->statusCode = Status::CODE_400.code;
      status_dto->message = "Invalid payload.";
      return createDtoResponse(Status::CODE_400, status_dto);
    }
```

Reply:
https://github.com/epsilla-cloud/vectordb/pull/50/commits/91883d9d3ad51f019445b989c911c62b476c209b